app/models/follower.rb

```diff
@@ -6,4 +6,15 @@ def initialize(follower_data)
     @handle = follower_data[:login]
     @url = follower_data[:html_url]
   end
+
+  def in_system?
+    @friend = User.find_by(handle: @handle)
+    @friend.present?
+  end
+
+  def already_friends?(user_id)
+    friendship = Friendship.find_by(friend_id: @friend.id, user_id: user_id)
+    friendship.present?
+  end
```

Comment:
This is only used for determining whether to display the button on the view for adding a friend